src/sub/swagger.yaml

```diff
@@ -301,6 +301,51 @@ paths:
       parameters:
         - $ref: '#/parameters/uidParam'
         - $ref: '#/parameters/subIdParam'
+    patch:
+      operationId: sub.payments.update_subscription
+      tags:
+        - Subscription
```

Comment:
Should be Subscriptions otherwise it is on a new node.

Reply:
fixed :)